RELEASING.rst

```diff
@@ -0,0 +1,55 @@
+Releasing sphinxcontrib-images
+==============================
+
+1. Test releasing to TestPyPI
+
+   - Bump the package version by editing ``sphinxcontrib/images.py``
+     (i.e. from 0.9.2 to 0.9.3.rc1 -- use .rcN for subsequent releases),
+     create a release branch with this change in the
+     https://github.com/sphinx-contrib/images repository, and finally create a
+     pull request.
+
+   - Trigger `publishing to TestPyPI via the GitHub UI`_ and select the release
+     branch as where to run the workflow.
+
+2. Test the TestPyPI release
+
+   - Make a new virtual environment::
+
+        read -p 'Version: ' VERSION \
+           && python3 -m venv venv-$VERSION \
+           && . venv-$VERSION/bin/activate
+
+   - Install the (pre-)release (see [1]_) and verify that the correct version
+     is installed::
+
+        pip install -U \
+              --index-url https://test.pypi.org/simple/ \
+              --extra-index-url https://pypi.org/simple \
+              "sphinxcontrib-images >= $VERSION.dev0" \
+           && pip freeze | grep sphinxcontrib-images
+
+   - Verify the pre-release package can build the package docs in docs/::
```

Comment:
"pre-release package" -> "release candidate package" if we are only using the `rcN`.